script/bootstrap

```diff
@@ -4,6 +4,13 @@ set -e
 
 cd "$(dirname "$0")/.."
 
+if [ -f "Brewfile" ] && [ "$(uname -s)" = "Darwin" ]; then
```

Comment:
Shouldn't we have something similar for Linux? After all, [our docs](https://github.com/github/linguist/blob/7855da1cf3f60731522b4fe392c358128e96edb9/CONTRIBUTING.md) include instructions for Linux users on setting up their environment too.

Reply:
@Alhadis I would love it if Linux had a standard for installing dependencies, but it varies greatly by distribution. We can't assume Linuxbrew either, as most folks won't have it installed.

Reply:
What do you mean by "standard"? Linux's package managers differ, but it should be enough to target the most popular ones.

Reply:
@Alhadis In the [scripts-to-rule-them-all](https://github.com/github/scripts-to-rule-them-all) pattern @rick references in this issue, we haven't historically accounted for various Linux distributions, but you're right there's no technical reason we couldn't. I would encourage you to open a new PR adding the proposed changes to support Linux and the maintainers can evaluate that on its own.